src/biogeochem/CNPhenologyMod.F90

```diff
@@ -907,13 +914,23 @@ subroutine CNSeasonDecidPhenology (num_soilp, filter_soilp       , &
                if (onset_gddflag(p) == 1.0_r8 .and. soilt > SHR_CONST_TKFRZ) then
                   onset_gdd(p) = onset_gdd(p) + (soilt-SHR_CONST_TKFRZ)*fracday
                end if
+               !separate into Arctic boreal and lower latitudes
+               if (onset_gdd(p) > crit_onset_gdd .and. abs(grc%latdeg(g))<45.0_r8) then
+                  onset_thresh=1.0_r8
+               else if (onset_gddflag(p) == 1.0_r8 .and. t10(p) > SHR_CONST_TKFRZ &
+                       .and. tmin10(p) > SHR_CONST_TKFRZ .and. ws_flag==1.0_r8 &
+                       .and. dayl(g)>(crit_dayl/2.0_r8) .and. snow5d(c)<0.1_r8) then
+                  onset_thresh=1.0_r8
+               end if
```

Comment:
Let's make sure we discuss this section Wednesday.

Reply:
As I say below having behavior completely different over a threshold is problematic numerically. It's better to have a smooth variation over a boundary. It's good to get them to at least be the same at the boundary, so for example make it piecewise-linear, even if the derivative is not. Having a jump at a boundary, might spell trouble with very high resolution simulations.

Reply:
Hi, this will be good to talk about Wednesday because I think my logic holds based on my understanding of what the phenology scheme was doing here. For my simulations, I did not need the latitudinal threshold because I was running a regional domain, so I added in the threshold as a possible way to do a global simulation.